src/components/status-bar/StatusBar.vue

```diff
@@ -43,34 +44,44 @@ export default {
   props: {
     portUsageDown: {
       type: Number,
+      default: 0,
     },
     portUsageUp: {
       type: Number,
+      default: 0,
     },
     packetError: {
       type: Number,
+      default: 0,
     },
     i2cError: {
       type: Number,
+      default: 0,
     },
     cycleTime: {
       type: Number,
+      default: 0,
```

Comment:
If we have defaults defined here, maybe we should remove these defaults in fc.js ??

Reply:
But i think is good to continue with defaults in fc.js and remove these defaults

Reply:
I'd keep both, this is just for component, it shouldn't care what's in `fc.js` or other files. There is logic which relies on this values inside of it.

We already have everything to tightly coupled. So relying on things which are in some global variable in random file is a pain in the neck to maintain as it is.